.github/template-bugReport.md

```diff
@@ -0,0 +1,18 @@
+**Issue Title:** Bug: [brief description of bug]
+
+Copy-and-paste the following content to the issue tracker and fill in with relevant details:
+```markdown
+**Tell us what environment you encountered the bug on, e.g. live server at `V5.93`, `master` branch at commit 1234567.**
```

Comment:
`Tell us what environment you encountered the bug on (e.g. live server at `V5.93`, `master` branch at commit 1234567)`

Suggest formatting it in this way.